### PR TITLE
Add task from theme html css javascript

### PR DESCRIPTION
**Pull request by IvelinYR**

### CODE REVIEW

File: html css javascript/form/main.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+        msgSucss.innerHTML = 'Success !'
+        msgSucss.style.color = 'green'
+    }else {
+        let msgError = document.getElementById(errorBox)
+        msgError.innerHTML = errorMessage
+        msgError.style.color = 'red'
+    }
+}
+
+//function TestField(name) {
```
Comment (mgenov) [inline html css javascript/form/main.js]: Commented code need to be removed.

File: html css javascript/form/main2.js [hunk 2]
```diff
@@ -0,0 +1,78 @@
+/**
+ * Created by clouway on 09.02.17.
+ */
+function validateUserForm () {
```
Comment (mgenov) [inline html css javascript/form/main2.js]: Outdated code need to be removed.

File: html css javascript/form/main.js [hunk 3]
```diff
@@ -0,0 +1,49 @@
+function ValidationField() {
```
Comment (mgenov) [inline html css javascript/form/main.js]: More looks like a function for the validation of the form: validateForm

File: html css javascript/calculator/main.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
+/**
+ * Created by clouway on 09.02.17.
+ */
+function validation(sing) {
```
Comment (mgenov) [inline html css javascript/calculator/main.js]: there is a typo: sign instead of sing

validateSign(value) as functions need to be verbs

File: html css javascript/calculator/main.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+    }
+    else if (check == '+'|| (check == '-')|| (check == '*') || (check == '/') ||(check == '.')){
+        return
+    }else
+        {
+            document.Calculator.display.value += sing
+    }
+}
+
+function point() {
```
Comment (mgenov) [inline html css javascript/calculator/main.js]: As I can see it handles pressing of dot. So it sounds reasonable to be named like: onDotPressed()

File: html css javascript/validationNumber/main.js [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+     if(isFractionValid(number)) {
+         document.getElementById('answer').innerHTML = 'Number is CORECT'
+         document.getElementById('answer').style.color = 'green'
+     }else{
+         document.getElementById('answer').innerHTML = 'Number is INCORECT'
+         document.getElementById('answer').style.color = 'red'
+     }
+}
+
+function isFractionValid(value) {
```
Comment (mgenov) [inline html css javascript/validationNumber/main.js]: You can add tests as they are shown in the docs.

Comment (IvelinYR) [inline html css javascript/validationNumber/main.js]: When I add tests in my code, I saw bug with negative number.
Bug is fixed.

File: html css javascript/form/main.js [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+        let msgSucss = document.getElementById(errorBox)
+        msgSucss.innerHTML = 'Success !'
+        msgSucss.style.color = 'green'
+    }else {
+        let msgError = document.getElementById(errorBox)
+        msgError.innerHTML = errorMessage
+        msgError.style.color = 'red'
+    }
+}
+
```
Comment (mgenov) [inline html css javascript/form/main.js]: these empty lines could be removed

File: html css javascript/calculator/main.js (hunk 4 above)
Comment (IvelinYR) [inline html css javascript/calculator/main.js]: renamed function validateSign(value)

File: html css javascript/calculator/main.js (hunk 5 above)
Comment (IvelinYR) [inline html css javascript/calculator/main.js]: renamed function  onDotPressed()

File: html css javascript/form/main.js (hunk 3 above)
Comment (IvelinYR) [inline html css javascript/form/main.js]: renamed  function validateForm ()

File: html css javascript/form/main.js (hunk 7 above)
Comment (IvelinYR) [inline html css javascript/form/main.js]: deleted empty lines

File: html css javascript/form/main.js (hunk 1 above)
Comment (IvelinYR) [inline html css javascript/form/main.js]: removed commented code

File: html css javascript/form/main2.js (hunk 2 above)
Comment (IvelinYR) [inline html css javascript/form/main2.js]: Removed outdated code

**Comment by mgenov** [comment]
One note: Please squash your changes before the review.
